src/gateway/kong-plugins/authentication/oidc/okta.md

```diff
@@ -113,16 +113,6 @@ guide, assume the route is in the default workspace.
 
 #### Minimum configuration requirements
 
-Configure the following parameters:
```

Comment:
@rajeevkong Since this content already existed (in some form) on each tab, I moved this info to the tabs instead of having it in three places and repeated.